nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
I figured that was the case. The error is here: https://travis-ci.org/nipy/nibabel/jobs/85392218#L977

Seems to be an immutability issue, but hard to diagnose, so this PR was me trying to use Travis to figure out if my guess was right.